### PR TITLE
Updating BuildAndTest.proj to manually deploy integration test dependencies.

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @jasonmalinowski, @jaredpar, @dotnet/roslyn-infrastructure 

This should hopefully help cut down the deployment issues for the open vs integration tests.

### CODE REVIEW

File: BuildAndTest.proj [hunk 1]
```diff
@@ -102,10 +95,20 @@
...
+    <Exec Condition="'$(TestVsi)' == 'true'" Command="&quot;$(VsixExpInstallerExe)&quot; $(VsixExpInstallerArgs) $(OutputDirectory)\Vsix\VisualStudioSetup\Roslyn.VisualStudio.Setup.vsix" />
+    <Exec Condition="'$(TestVsi)' == 'true'" Command="&quot;$(VsixExpInstallerExe)&quot; $(VsixExpInstallerArgs) $(OutputDirectory)\Vsix\VisualStudioSetup.Next\Roslyn.VisualStudio.Setup.Next.vsix" />
+    <Exec Condition="'$(TestVsi)' == 'true'" Command="&quot;$(VsixExpInstallerExe)&quot; $(VsixExpInstallerArgs) $(OutputDirectory)\Vsix\VisualStudioInteractiveComponents\Roslyn.VisualStudio.InteractiveComponents.vsix" />
+    <Exec Condition="'$(TestVsi)' == 'true'" Command="&quot;$(VsixExpInstallerExe)&quot; $(VsixExpInstallerArgs) $(OutputDirectory)\Vsix\ExpressionEvaluatorPackage\ExpressionEvaluatorPackage.vsix" />
+    <Exec Condition="'$(TestVsi)' == 'true'" Command="&quot;$(VsixExpInstallerExe)&quot; $(VsixExpInstallerArgs) $(OutputDirectory)\Vsix\VisualStudioDiagnosticsWindow\Roslyn.VisualStudio.DiagnosticsWindow.vsix" />
```
Comment (jasonmalinowski) [inline BuildAndTest.proj]: Just to confirm: the integration test support binary is in the diagnostics window VSIX?

Comment (tannergooding) [inline BuildAndTest.proj]: No. I missed the integration test vsix 😄

File: BuildAndTest.proj [hunk 2]
```diff
@@ -102,10 +95,21 @@
       <CoreRunArgs>$(CoreClrTestDirectory)\xunit.console.netcore.exe @(CoreTestAssemblies, ' ') -parallel all -xml $(CoreClrTestDirectory)\xUnitResults\TestResults.xml</CoreRunArgs>
       <RunTestsExe>Binaries\$(Configuration)\Exes\RunTests\RunTests.exe</RunTestsExe>
       <RunTestsArgs>$(NuGetPackageRoot)\xunit.runner.console\$(xunitrunnerconsoleVersion)\tools $(RunTestArgs) @(TestAssemblies, ' ')</RunTestsArgs>
+      <VsixExpInstallerExe>$(NuGetPackageRoot)\roslyntools.microsoft.vsixexpinstaller\$(RoslynToolsMicrosoftVSIXExpInstallerVersion)\tools\VsixExpInstaller.exe</VsixExpInstallerExe>
```
Comment (jaredpar) [inline BuildAndTest.proj]: Thought you were going ot push for using the VS based version of the tool.  Did something change?

Comment (tannergooding) [inline BuildAndTest.proj]: It requires admin privileges to deploy to a hive other than the default, so we can't use it. They have a bug tracking it.

**Pull request merged**